TESTING.md

```diff
@@ -19,4 +19,4 @@ The project runs two Kubernetes clusters on which all e2e tests are ran automati
 
 Both the cluster management as well as the cloud resources required for our automated tests are managed by Terraform and [available on GitHub](https://github.com/kedacore/testing-infrastructure) so that every contributor can open a PR with the infrastructure changes that they require. Everything is automatically deployed by using GitHub Actions to ensure we are running the latest configuration and can easily migrate to other infrastructure, if we have to.
 
-Additionally, CNCF sponsors KEDA by providing arm64 machines on which we build our ARM64 image and Vexxhost sponsors an OpenStack instance to run tests on these as well.
+Additionally, Vexxhost sponsors an OpenStack instance to run tests.
```

Comment:
CNCF stills sponsors KEDA (we have ARM machines but also cloud credits) so I'd not remove this